app/scrapers/mojeek.py

```diff
@@ -27,3 +28,23 @@ def parse_response(soup):
         print('Mojeek parsed: ' + str(urls))
 
         return urls
+
+    @staticmethod
+    def parse_news_response(soup):
```

Comment:
Just a suggestion, wouldn't it be a better option to just reuse parse_response, why add redundant code

Reply:
No for more simplified code, we have to stick to making a new function only.

Reply:
We should look into what @dilraj45 said. I don't think it'll make it very complex, here is how I'd go about it

The response parse function while parsing links, attaches a `type` field to item and returns all results, be it video, news or anything.
Now we just need to filter the ones we need. I think this would be more elegent than creating different function for each type of results we want.
Suggestions ?

Reply:
So, this is a condition for a feature to add, or your own concern?
The way you both are suggesting is a complex one and through that way we will increase the complexity https://github.com/fossasia/query-server/blob/master/.travis.yml and nothing else.

Secondly, its not necessary that only after making the change that you suggested we will be able to add the feature. Though its working properly in the link https://gentle-basin-55487.herokuapp.com/
If this link is not working properly, then Please let me know.

Reply:
I am not saying it's not working, it's about writing good code.

Reply:
Please search about the complexity attribute at travis Ci . 
We are going to increase flake8 complexity in the manner you are suggesting.
I have tried doing that in Parsijoo Image/Video PR. That's what I am saying. 
Additional if else increases complexity in travis

Reply:
I agree @bhaveshAn, that adding if-else will increase the cyclomatic complexity, but I guess we need to decide upon an acceptable term between better coding practices and cyclomatic complexity of the code. Feel free to take your call. Additionally, for this particular engine, the implementation of parse_response and parse_news_response is exactly same, so you can at least,  instead of rewriting the whole implementation for parse_news_response, just make a call for parse_response in parse_news_response. This will reduce the boilerplate code, plus testing will be easier. Let me know what are your thoughts about this :smile: